webrtc/historical.html

```diff
@@ -23,4 +23,8 @@
     assert_false(name in RTCPeerConnection.prototype);
   }, "RTCPeerConnection member " + name + " should not exist");
 });
+
+test(function() {
+  assert_false("webkitRTCPeerConnection" in window);
+}, "webkitRTCPeerConnection interface should not exist");
```

Comment:
Since you've add `mozGetUserMedia` in mediacapture-streams/historical.html, should this file also include: 

```
mozRTCIceCandidate
mozRTCPeerConnection
mozRTCSessionDescription
```

?

Reply:
Yes, for sure! I didn't know they existed :)